lib/libc/regex/regcomp.c

```diff
@@ -329,7 +329,7 @@ regcomp_internal(regex_t * __restrict preg,
 		computejumps(p, g);
 		computematchjumps(p, g);
 		if(g->matchjump == NULL && g->charjump != NULL) {
-			free(g->charjump);
+			free(&g->charjump[CHAR_MIN]);
```

Comment:
This change matches the `&g->charjump[-(CHAR_MIN)]` when allocated (though stylistically I'd rather those were `g->charjump +/- CHAR_MIN` to be clear they're offsetting rather than trying to refer to a single element, even if semantically the two forms are defined to be equivalent).